README.rst

```diff
@@ -105,6 +105,7 @@ The following parameters are optional:
 - :code:`rp_hierarchy_dirs_level = 0` - Directory starting hierarchy level (from pytest.ini level) (default `0`)
 - :code:`rp_hierarchy_dirs = True` - Enables hierarchy for tests directories, default `False`. Doesn't support 'xdist' plugin.
 - :code:`rp_hierarchy_dir_path_separator` - Path separator to display directories in test hierarchy. In case of empty value current system path separator will be used (os.path.sep)
+- :code:`rp_hierarchy_code` - Enables hierarchy for parametrized tests, default `False`
```

Comment:
Not only for psrameterized tests, but also for inner classes.

Reply:
updated